stacks/apps/augury.yml

```diff
@@ -534,6 +556,34 @@ Resources:
       Statistic: Sum
       Threshold: 1
       TreatMissingData: breaching
+  AuguryJobsProducedErrorsAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub ERROR [Augury] Worker <${EnvironmentTypeAbbreviation}> IS THROWING THE ERRORS (${RootStackName})
+      AlarmDescription: !Sub >-
+        ${EnvironmentType} Augury's worker task has raised an error. Check the logs!
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 3
+      MetricName: !GetAtt Constants.JobErrorsCountMetricName
+      Namespace: !Ref kMetricFilterNamespace
+      Period: 300
+      Statistic: Sum
+      Threshold: 1
+      TreatMissingData: notBreaching
+  AuguryExpandedAllocationCacheSizeAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub ERROR [Augury] Allocation Targeting Record Cache <${EnvironmentTypeAbbreviation}> IS LARGE (${RootStackName})
+      AlarmDescription: !Sub >-
+        ${EnvironmentType} Augury's expanded allocation cache is big for some flight(s). Check the logs and the TruncateExpiredDataJob job.
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 3
+      MetricName: !Sub Constants.TargetingRecordCacheSizeMetricsName
+      Namespace: !Ref kMetricFilterNamespace
+      Period: 300
+      Statistic: Maximum
+      Threshold: 1000
+      TreatMissingData: notBreaching
   AuguryActualsAreBehindAlarmTags:
     Type: Custom::CloudWatchAlarmTags
     Condition: IsProduction
```

Comment:
Also noting that these tags are conditional, but the alarm itself isn't. I know it's unrelated to this PR, but if you wanted to remove this condition that would be good.